travis-test

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
 set -ev
-eslint .
-nyc npm test && nyc report --reporter=text-lcov | coveralls
+npm install
+npm run coveralls
```

Comment:
npx is cool, but I was going for something slightly different here. I wanted it all encapsulated in the package.json. I've run into some issue with test scripts getting out of date and this way we can avoid it.

Reply:
@hweeks That is fine. Still not sure I'm a fan of how this is setup tho. I saw you had a commit that got reverted that did something similar to this.

I think my biggest problem with this is the fact that the coveralls command runs both the CI command and the coveralls portion. I feel like we should have a coveralls script that only handles that portion.

What id we run `npm run ci && npm run coveralls` here. Then have our coveralls script be like: `nyc report --reporter=text-lcov | coveralls`?

Reply:
the coveralls script would fail if run standalone. `nyc report` relies on a generated `.nyc_output`. i generally try to prevent scripts that can be a no-op at any point. does that make any sense?

Reply:
@hweeks Yeah that makes sense. Hmm. Let's just rename this to be `coveralls-ci` (or something similar) for now and go with it then.

Reply:
updated